wgsl/index.bs

```diff
@@ -1619,6 +1627,65 @@ texel_format
 
 </pre>
 
+## Atomic Types ## {#atomic-types}
+
+<table class='data'>
+  <thead>
+    <tr><th>Type<th>Description
+  </thead>
+  <tr><td>atomic<*T*><td>Atomic of type *T*. *T* must be either [=u32=] or [=i32=].
+</table>
+
+Operations on atomic objects in [SHORTNAME] are mutually ordered for each
+object. That is, multiple invocations may modify the object without causing a
```

Comment:
We haven't defined data-race.

Suggest this instead:

> Operations on atomic objects in [SHORTNAME] are mutually ordered for each object.
> That is, during execution of a shader stage, for each atomic object A, all agents observe the same order of operations applied to A.
> The ordering for distinct atomic objects may not be related in any way; no causality is implied.
> Note that variables in [=storage class/workgroup=] storage are shared within a [=compute shader stage/workgroup=], but are not shared between different workgroups.

I wondered about whether to make this a definition for "mutually ordered", but I think we should wait on that until we add the memory model.  In Vulkan mutually ordered is a consequence of more basic conditions.